index.xml

```diff
@@ -12005,12 +11827,51 @@ interface
                     Try running the following steps:
                     <ol>
                       <li>
+                        Let <var>O</var> be <span class='esvalue'>null</span>.
+                      </li>
+
+                      <li>
+                        If <var>target</var> is not a <a class='dfnref'
+                        href='#dfn-namespace'>namespace</a>, and <var>op</var> is not a <a
+                        class='dfnref' href='#dfn-static-operation'>static operation</a>:
+
+                        <ol>
+                          <li>
+                            If the <span class='esvalue'>this</span> value is <span
+                            class='esvalue'>null</span> or <span class='esvalue'>undefined</span>,
+                            set <var>O</var> to <var>realm</var>'s <a class='dfnref external'
+                            href='https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-global'>global
+                            object</a>. (This will subsequently cause a <span
+                            class='estype'>TypeError</span> in a few steps, if <var>target</var> is
+                            not the appropriate global object.)
+
+                            <!-- https://www.w3.org/Bugs/Public/show_bug.cgi?id=18547#c9 -->
+                          </li>
+
+                          <li>Otherwise, set <var>O</var> to the <span class='esvalue'>this</span>
+                          value.</li>
+
+                          <li>If <var>O</var> is a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a>, then <a class='dfnref'
+                          href='#dfn-perform-a-security-check'>perform a security check</a>,
+                          passing <var>O</var>, <var>realm</var>, <var>id</var>, and "method".</li>
+
+                          <li>If <var>O</var> is not a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a> that implements the
+                          interface <var>target</var>, <a href='#ecmascript-throw' class='dfnref
```

Comment:
This is not quite right, depending on what it means to "implement" an interface.  The current spec text is careful to ensure that `I` is an actual interface, not a mixin (except for the weirdness around globals, wheere I don't understand what the current spec text is doing and why).  I think we should fix the caller of this algorithm to pass in the "interface whose interface prototype object (or interface object, for a static operation) this property corresponding to the operation appears on" to preserve the current semantics.  At _that_ point calling it `target` would make sense, actually.